### ISSUE TITLE
[V3] Userinfo..

### ISSUE THREAD

**Issue by Ray-MRQ** [Cog-Creators/Red-DiscordBot#1992]
```py
Exception in command 'userinfo'
Traceback (most recent call last):
  File "/home/ramimqr/Red-V3/venv/lib/python3.6/site-packages/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/home/ramimqr/Red-V3/venv/lib/python3.6/site-packages/redbot/cogs/mod/mod.py", line 1279, in userinfo
    names, nicks = await self.get_names_and_nicks(user)
  File "/home/ramimqr/Red-V3/venv/lib/python3.6/site-packages/redbot/cogs/mod/mod.py", line 1365, in get_names_and_nicks
    names = [escape(name, mass_mentions=True) for name in names]
  File "/home/ramimqr/Red-V3/venv/lib/python3.6/site-packages/redbot/cogs/mod/mod.py", line 1365, in <listcomp>
    names = [escape(name, mass_mentions=True) for name in names]
  File "/home/ramimqr/Red-V3/venv/lib/python3.6/site-packages/redbot/core/utils/chat_formatting.py", line 315, in escape
    text = text.replace("@​everyone", "@\u200beveryone")
AttributeError: 'NoneType' object has no attribute 'replace'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/ramimqr/Red-V3/venv/lib/python3.6/site-packages/discord/ext/commands/bot.py", line 886, in invoke
    yield from ctx.command.invoke(ctx)
  File "/home/ramimqr/Red-V3/venv/lib/python3.6/site-packages/discord/ext/commands/core.py", line 514, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/home/ramimqr/Red-V3/venv/lib/python3.6/site-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'NoneType' object has no attribute 'replace'
```

There are more errors but I am unable to retrieve them as well as the fact that I had to keep deleting my settings file to get it to work which is pointless if there is nothing stored in them when it does work.

After a while the settings file just says "null"... 
https://pastebin.com/wk88pzJs

**Comment by mikeshardmind** [comment]
This was fixed in #1865 , please update your instance.

**Comment by Ray-MRQ** [comment]
Aha ok sorry
